### PR TITLE
bundle missing fonts

### PR DESCRIPTION
**Pull request by Lctrs**

### CODE REVIEW

**Comment by gasinvein** [comment]
I think this is not the right approach. You're effectively building parts of Proton-GE outside flathub infra, which is out of scope of producing source tarball.

**Comment by Lctrs** [comment]
Is this a problem for font files ?

**Comment by gasinvein** [comment]
I'd say it is for any source->binary transition. Also there might be license issues, since Libreation fonts are under GPL which requires to provide source for binaries.
At least you should pack prebuilt fonts separately, so that the source tarball contains full upstream source and only it.

**Comment by Lctrs** [comment]
@gasinvein better ?

**Comment by gasinvein** [comment]
@Lctrs Better, especially if it works :)
I've failed to make afdko work for Proton and delayed bundling fonts for future.

**Comment by Lctrs** [comment]
@gasinvein It builds successfully and the fonts look good in `org.gnome.FontManager` at least :)

File: com.valvesoftware.Steam.CompatibilityTool.Proton.yml [hunk 1]
```diff
@@ -75,6 +76,60 @@ cleanup:
...
+          - -DENABLE_WOFF2=OFF
+          - -DENABLE_DOCS=OFF
+        cleanup:
+          - "*"
+        sources:
+          - type: archive
+            url: "https://github.com/fontforge/fontforge/releases/download/20201107/fontforge-20201107.tar.xz"
+            sha256: 68bcba8f602819eddc29cd356ee13fafbad7a80d19b652d354c6791343476c78
+      - files/fonts/afdko-dev.json
+      - files/fonts/afdko.json
```
Comment (gasinvein) [inline com.valvesoftware.Steam.CompatibilityTool.Proton.yml:130]: My approach was to name additional files by the upstream project name they're related to. I.e. these would be something like `files/afdko/python3-setuptools_scm.json` and `files/afdko/python3-afdko.json`. But that's just a personal preference.

Comment (Lctrs) [reply com.valvesoftware.Steam.CompatibilityTool.Proton.yml:130]: I've named them after `requirements.txt` and `requirements-dev.txt` in `afdfko` source tree.

Comment (gasinvein) [reply com.valvesoftware.Steam.CompatibilityTool.Proton.yml:130]: Maybe `files/afdko/python3-requirements-dev.json` and `files/afdko/python3-requirements.json`, then?

Comment (Lctrs) [reply com.valvesoftware.Steam.CompatibilityTool.Proton.yml:130]: I'm okay to rename the files, but I'm reluctant to redo a full build just for this. Is this okay if I cancel the test build and just merge this given that the previous build worked ?

Comment (gasinvein) [reply com.valvesoftware.Steam.CompatibilityTool.Proton.yml:130]: I guess so, if you don't plan to add any more changes.

Comment (Lctrs) [reply com.valvesoftware.Steam.CompatibilityTool.Proton.yml:130]: No more change, from my POV, it's all good.

**Comment by gasinvein** [comment]
Why did you put fonts to the beginning of the modules list, BTW? Is that beacause of flatpak/flatpak-builder#282?

**Comment by Lctrs** [comment]
@gasinvein Yes.

**Comment by gasinvein** [comment]
Maybe put them after "heavy" modules, but add more workaround symlinks instead? I think `files/afdko/files -> ..` should work.

**Comment by Lctrs** [comment]
Do you know if they're required to build the default prefix ?

**Comment by gasinvein** [comment]
The default prefix contains symlinks to them. Not sure if their presence is required during it's creation, though. But probably.

**Comment by Lctrs** [comment]
Let's see.

**Pull request merged**